Core/Inc/DynamicControls/lateral_control.h

```diff
@@ -22,12 +22,18 @@
 #define Y1 (0.599520598 + 0.523598776)/2
 #define X0 -1.941983885
 #define X1 1.941983885
+//defines PID LATERAL
+#define KP_LATERAL 6.51864262048678
+#define KI_LATERAL 43.9160892044863
+#define TI_LATERAL KP_LATERAL / KI_LATERAL //0.14843404179579
```

Comment:
Coloca entre parenteses, boa prática pra evitar erros.
```suggestion
#define TI_LATERAL (KP_LATERAL / KI_LATERAL) //0.14843404179579
```